edkrepo/commands/combo_command.py

```diff
@@ -47,13 +47,13 @@ def run_command(self, args, config):
             combo_list.append(manifest.general_config.current_combo)
         for combo in sorted(combo_list):
             if combo == manifest.general_config.current_combo:
-                print("* {}{}{}".format(Fore.GREEN, combo, Fore.RESET))
+                ui_functions.print_info_msg("current combo: " + combo, header=False)

```

Comment:
This print was meant to emulate the formatting of git branch where the current branch is displayed in green with an asterisk. While this is a deviation from the standardized messaging lets retain that formatting for the current combo and use the previous formatting for this line.